pages/jdl.md

```diff
@@ -51,8 +51,8 @@ You can use it by:
 
 and *Voilà*, you are done!
 
-If you work in a team, perhaps you would like to have multiple files instead of one. We added this option so that you don't manually 
-concatenate all the files into one, you just have run `jhipster-uml my_file1.jh my_file2.jh` or `jhipster:import-jdl my_file1.jh my_file2.jh`.
+If you work in a team, perhaps you would like to have multiple files instead of one. We added this option to JHipster UML so that you don't manually
```

Comment:
This works

Le 5 avril 2016 17:27:11 GMT+02:00, Deepu K Sasidharan <notifications@github.com> a écrit :
>> @@ -51,8 +51,8 @@ You can use it by:
>>  
>>  and *Voilà*, you are done!
>>  
>> -If you work in a team, perhaps you would like to have multiple files
>instead of one. We added this option so that you don't manually 
>> -concatenate all the files into one, you just have run `jhipster-uml
>my_file1.jh my_file2.jh` or `jhipster:import-jdl my_file1.jh
>my_file2.jh`.
>> +If you work in a team, perhaps you would like to have multiple files
>instead of one. We added this option to JHipster UML so that you don't
>manually
>
>@MathieuAA Im removing this for now as it will not work in the sub
>generator, if we are able to fix it before next release ill update this
>PR accordingly
>
>---
>You are receiving this because you were mentioned.
>Reply to this email directly or view it on GitHub:
>https://github.com/jhipster/jhipster.github.io/pull/250/files/f95b710abc3388d0491fca289730ae314ecdc390#r58559171

-- 
Mathieu ABOU-AICHI